### DISCUSSION TITLE
Using Vector as a proxy for Datadog agents

### DISCUSSION THREAD

**Discussion by lognarly** [vectordotdev/vector#10024] [Q&A] [answered]
Can Vector be used as an endpoint for filtering/proxying metrics from Datadog agents? I have customers that don't have proper proxies but want to utilize Datadog agents to still collect metrics from systems with no outbound internet access. Can Vector be used as a metric proxy for systems with Datadog agents that have no outbound internet access?

**Comment by jszwedko** [comment, marked as answer]
Hi @lognarly !

Soon! We are actively working in ingesting metrics from the Datadog Agent in Vector. Once that is done, you'll be able to pass the metrics from the Datadog Agents through Vector to Datadog.

There is an open pull request adding support here: https://github.com/vectordotdev/vector/pull/9563

**Comment by atibdialpad** [comment]
I use the Vector statsd source to listen to datadog agent's dogstatsd and then output event to dd via the datadog_metric_sink.
